vulnerability-report.md

```diff
@@ -0,0 +1,55 @@
+# A1 - Injection
```

Comment:
Your vulnerability report doesn't really follow any of the format outlined in the sample report. Please reformat and update.